src/components/elements/wrapper/wrapper.tsx

```diff
@@ -112,6 +114,18 @@ export const Wrapper = (props: IWrapperProps): JSX.Element | null => {
 
 		const warning = warnings?.[childId];
 
+		if ("colProps" in childSchema) {
+			return (
+				<Layout.ColDiv
+					data-testid={TestHelper.generateId(childId, "grid_item")}
+					desktopCols={12}
+					{...childSchema.colProps}
+				>
+					<FieldWrapper id={childId} schema={childSchema} Field={Field} />
+					{warning && <DSAlert type="warning">{warning}</DSAlert>}
+				</Layout.ColDiv>
+			);
+		}
```

Comment:
a similar implementation is also applied in design-system v2.0.0-canary.5, however let's discuss this implementation because we may still need this but in another PR